mta-sts.md

```diff
@@ -498,4 +498,34 @@ https://mta-sts.example.com/.well-known/mta-sts.json:
 
 ~~~~~~~~~
 
+# Appendix 2: Sending MTA State Machine
+
+It is helpful to think of the different states of a sending MTA and a receiving
```

Comment:
I'm late to the party, but it does feel hard to read. Trying simpler phrasing:

**Case 1: Sender has an unexpired "enforce" policy in its cache:** 
   a. If the policy _has_ been applied before, this implies messages will only be delivered to an MX that validates according to the policy unless and until a new policy is served.
   b. If the policy _has not_ been applied before, this implies messages _can_ be delivered to an MX that does not validate according to the cached policy.

**Case 2: Sender had multiple, unexpired policies in its cache:**
   a. If the cache contains a `report`-mode policy that _was_ successfully applied before, along with an `enforce`-mode that was _not_ successfully applied before, this implies that messages _can_ be delivered to an MX that does not validate according to the cached policy.

WDYT?